tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/FeverBracesSuite.scala

```diff
@@ -0,0 +1,236 @@
+package scala.meta.tests.parsers.dotty
+
+import scala.meta.tests.parsers._
+import scala.meta._
+import scala.meta.tests.tokenizers.TokenizerSuite
+
+class FeverBracesSuite extends BaseDottySuite {
+
+  override implicit val dialect = dialects.Scala3.withAllowFewerBraces(true)
```

Comment:
> I'm wondering if we are going to enable it by default for scala3.

We might, but I was thinking a safer approach would be to do it gradually.

> Otherwise we will have to add new options to scalafmt settings.

We can already change a single dialect options thanks to the changes by @kitbellew 

> Also, as it might be enabled by import we would need the same mechanic

We can enable it by default for nightly versions first and change it to stable later on.

Reply:
> We can already change a single dialect options thanks to the changes by @kitbellew

oh, cool. didn't know that!

Then we would need only import mechanic but we can do it separate PR

Reply:
I mean the case when you parse the source using std Scala3 dialect and if it contains:
`import import language.experimental.fewerBraces` then parser treats  it as  it was called with `Scala3.withAllowFewerBraces(true)` dialect